refresh.template.py

```diff
@@ -683,7 +694,36 @@ def _get_apple_active_clang():
     # Unless xcode-select has been invoked (like for a beta) we'd expect, e.g., '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang' or '/Library/Developer/CommandLineTools/usr/bin/clang'.
 
 
-def _apple_platform_patch(compile_args: typing.List[str]):
+@functools.lru_cache(maxsize=None)
+def _get_apple_active_swiftc():
+    """Get path to xcode-select'd swiftc version."""
+    return subprocess.check_output(
+        ('xcrun', '--find', 'swiftc'),
+        stderr=subprocess.DEVNULL, # Suppress superfluous error messages like "Requested but did not find extension point with identifier..."
+        encoding=locale.getpreferredencoding()
+    ).rstrip()
+
+
+def _apple_swift_patch(compile_args: typing.List[str]):
+    """De-Bazel(rule_swift) the command into something sourcekit-lsp can parse."""
+
+    # Remove build_bazel_rules_swift/tools/worker/worker
+    compile_args.pop(0)
+
+    # Expand -debug-prefix-pwd-is-dot
+    match = next((i for i,v in enumerate(compile_args) if v == "-Xwrapped-swift=-debug-prefix-pwd-is-dot"), None)
+    if match:
+        compile_args[match] = "-debug-prefix-map"
+        compile_args.insert(match + 1, os.environ["BUILD_WORKSPACE_DIRECTORY"] + "=.")
```

Comment:
Like with clang, might be able to just strip this out, since the commands are conceptional happening in place, not that we're actually running/debugging with them anyway.

Reply:
Those flags look handy for configuring sourcekit-lsp! (like in https://github.com/hedronvision/bazel-compile-commands-extractor/pull/89#discussion_r1037779449)

But I just meant that it might make sense just omit that particular flag, -debug-prefix-map, since I don't think it'll impact analysis--nor compilation if it's just the cwd.

Reply:
Done
https://github.com/xinzhengzhang/bazel-compile-commands-extractor/commit/e96bbbee80feee3425e7fb6e510bc20d430619be